serverless/knative_eventing/serverless-channels.adoc

```diff
@@ -1,13 +1,13 @@
 include::modules/serverless-document-attributes.adoc[]
 [id="serverless-channels"]
-= Using channels
+= Using channels with Knative Eventing
```

Comment:
This will all be overwritten by https://github.com/openshift/openshift-docs/pull/23815 anyway, so either rebase that on top of this or vice versa. This section shouldn't take up much review time since it's changing anyway, IMHO.